src/PresentationalComponents/Battery/index.js

```diff
@@ -0,0 +1 @@
+export {default as Battery } from './battery';
```

Comment:
This is great way of exporting components! Please add `export * from './PresentationalComponents/Battery';` into `src/index.js` as well so consumers can import this component either calling `import { Battery } from '@red-hat-insights/insights-frontend-components/components/Battery';` or simply `import { Battery } from '@red-hat-insights/insights-frontend-components';`